onnxruntime/core/providers/qnn/builder/opbuilder/base_op_builder.cc

```diff
@@ -45,54 +46,6 @@ Status BaseOpBuilder::AddToModelBuilder(QnnModelWrapper& qnn_model_wrapper,
   return Status::OK();
 }
 
-bool BaseOpBuilder::OnnxDataTypeToQnnDataType(const int32_t onnx_data_type, Qnn_DataType_t& qnn_data_type, bool is_quantized) const {
```

Comment:
Moved this and GetQnnDataType() to a common utils file so that we can access from outside the BaseOpBuilder hierarchy.